DC/Lab10/EOSC350Lab.cls

```diff
@@ -0,0 +1,153 @@
+%==============================
```

Comment:
can you please remove this from here: the pdf should only be in the private eosc350 repo and on the eosc350website